### PR TITLE
Add object and feature LiveStatusListener

### LINKED ISSUE #17: Add object and feature LiveStatusListener

### PR DESCRIPTION
**Pull request by martialblog**
- Fixes #17

### CODE REVIEW

File: plugins/modules/icinga2_livestatuslistener.py [hunk 1]
```diff
@@ -0,0 +1,29 @@
+from ansible.module_utils.basic import AnsibleModule
```
Comment (martialblog) [inline plugins/modules/icinga2_livestatuslistener.py:1]: TODO need to add docs

**Review by mkayontour — CHANGES_REQUESTED** [review]
Docs for the feature are missing, other than that looks good.

File: plugins/modules/icinga2_livestatuslistener.py (hunk 1 above)
Comment (martialblog) [reply plugins/modules/icinga2_livestatuslistener.py:1]: Rebased the Branch and added Docs

**Comment by martialblog** [comment]
Updated the Docs. Tested the Feature, fixed some final stuff. 

I think we're good to go.

**Review by mkayontour — APPROVED** [review]

**Pull request merged**